### ISSUE TITLE
add a note to clarify "voters"

### ISSUE THREAD

**Issue by AngelaKTE** [TE-Academy/Reputation-based_Weighted_Voting#51]
In all evaluation metrics, we require dictionaries with voters (and weights).<br>Do we agree that “voters” refer to wallets holding one or several NFTs?<br>If yes, it might be worth clarifying.<br><br>It’s valid to put the verification metrics in context to “voters” rather than NFTs.<br>Because this is all evaluating the risk of weighting, and the risk is mainly in voter behavior, not in a technical setting of NFTs.<br>So I agree, let's highlight this aspect in the demo and writeup.

**Comment by eightarmsninebrains** [comment]
I agree that this is the definition of voter: a wallet/account holding various NFTs. 

Please feel free to add this note anywhere you like. :)
